examples/legacy/not_checked/simple/form.rb

```diff
@@ -4,7 +4,9 @@
   background "../static/menu-left.png", :top => 50, :width => 55
   background "../static/menu-right.png", :right => 0, :top => 50, :width => 55
   image "../static/menu-corner1.png", :top => 0, :left => 0
-  image "../static/menu-corner2.png", :right => 0, :top => 0
+  # image "../static/menu-corner2.png", :right => 0, :top => 0
```

Comment:
Looking pretty good! Do we want to resolve the commented code?

Reply:
yeah image dont work with local files as of now ig? tho its not showing any error 

that's the reason i haven't moved that file to working directory yet

Reply:
and ig we need this 

https://github.com/shoes/shoes3/tree/master/static

folder into our branch for many examples ig for images

Reply:
oh nvm it is in docs/static directory